samples/word/Basics/basic-doc-assembly.yaml

```diff
@@ -0,0 +1,241 @@
+id: ciyvyru7l00014r6ydkzkwka7
+gist: ''
+source: WORD
+author: Juan Balmori - Microsoft
+name: Basic Word APIs

```

Comment:
Consider calling this "Basic document assembly", if it better describes the theme of the snippet